### DISCUSSION TITLE
fipple factor: temp offset for entering freq. into no-holes-tuning file

### DISCUSSION THREAD

**Discussion by JeffBurris** [edwardkort/WWIDesigner#108] [NAF Study] [answered]
Hello...
To make flutes good for the 72F U.S. standard, I leave the temperature in WID's options at 72F.
I then change the settings on my tuning app to offset for the real temperature in my workshop.
This works fine while I'm looking at notes and how sharp and flat they are.
But in regard to fipple factor procedure if I understand it correctly:
In the no-holes tuning file I use to help determine my fipple factor, I am supposed to enter the actual fundamental frequency my flute is blowing. To simplify let's say I blew a 441 and I didn't trim any further because it will flatten out later with more holes.
But, because of the way I'm doing this, since my customers and WID are all set for 72F
I should tell the no holes tuning file what I *would* blow at 72F.
Because it's colder than 72F in my room, I should manually adjust or offset that frequency that I tell WID in that one field, not enter what frequency I blew.
* The reason I'm saying this is because no matter what I offset my tuning app for, that only ever affects what it tells me about letter-notes and how sharp or flat they are (which is great for the rest of the tuning process)
-- BUT the frequency my tuning app tells me is always, always the real frequency it hears -- not some offset affected by whatever I set A to equal in the tuning app. I can set that A to 440, or 500, or 400 -- doesn't matter -- it always tells me I'm blowing about a 441 for this one flute's fundamental I stopped trimming at.
So for this one operation (telling the no holes tuning files what fundamental frequency my flute is really blowing), if it's colder in my room than 72F, I need to look at a chart manually and adjust that 441 Hz I really just blew to... 442 or whatever correct offset applies. So long as I chose to keep WID set to 72F and do it that way.
The confusing part I think can be because the no holes tuning file is not asking me how flat or sharp my fundamental frequency is blowing -- it asks what the numeric frequency is -- and that never gets offset by the tuning app.

Thank you in advance and thanks always
Jeff

**Comment by edwardkort** [comment, marked as answer]
Hi Jeff,

In the best of all possible worlds, the fipple factor is temperature independent. That is, when using the optimizer to determine fipple factor:
- In the Options/Preferences dialog, set the temperature to the actual temperature at which you are measuring the flute.
- When measuring the frequency of the no-holes flute, make no corrections. Use what the tuner tells you for frequency.

Then when you do further hole-layout design, change the Options/Preferences value back to 72F (although it only accepts values in C).

The reason for the caveat is that the temperature of the air in the flute bore is not exactly ambient - it is heated somewhat by the higher temperature of your breath. Clint Goss did the experiment to see how much the tuning changes. I believe that the paper can still be found on his web site, flutopedia.com. And your results may be different than Clint's; mine were.

My earlier program, NAFlutomat, allows you to independently set the construction temperature (tuning temp) and ultimate playing temperature (playing temp) and then it does the trivial transformation between the two. I didn't want to complicate the WIDesigner interface. I figured the user could apply the 1.7 cents/degree F correction outside of the program.

If you want to confirm/refute the above, I love an experiment. Take your no-holes flute, play it at at one temperature and calculate the fipple factor "for" that temperature. Then do the same thing at a different temperature (like in your shop and in your home). The two fipple factors should be very close to the same. Please let us know how it turns out.

Cheers,
Edward Kort
Wood Wind

**Reply by JeffBurris** [reply]
Ok thanks so much -- very comfortable sanity check, and also confirms how I can use the temps in Options/Preferences separately between the fipple factor determination process and the actual build, instead of needing to lie to it manually during the fipple factor determination process.

I noticed the construction and playing temperature feature in NAFlutomat vs. WID, but I can just set my tuner for that.

Oh I had seen, just earlier, the paper about the said, added factors of the temp topic, at Flutopedia... offsets for the offsets, hehe; whatever works!
Precision and accuracy are seldom simple.
Thanks again
